EventsExpress.Db/Migrations/AppDbContextModelSnapshot.cs

```diff
@@ -127,6 +127,14 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<Guid>("CityId")
                         .HasColumnType("uniqueidentifier");
 
+                    b.Property<Guid>("CreatedBy")
+                        .HasColumnType("uniqueidentifier");
+
+                    b.Property<DateTime>("CreatedDateTime")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("datetime2")
+                        .HasDefaultValue(new DateTime(2020, 11, 21, 11, 35, 55, 387, DateTimeKind.Utc).AddTicks(5144));
```

Comment:
This default value is always set to when you created this DateTime object. You should set it to `DateTime.UtcNow` or `GETUTCDATE` function as in here (https://docs.microsoft.com/en-us/sql/t-sql/functions/getutcdate-transact-sql?view=sql-server-ver15).